mobile/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,12 @@
             </intent-filter>
         </activity-alias>
 
+        <!-- "AR Optional" app, contains non-AR features that can be used when
+        "Google Play Services for AR" (ARCore) is not available. -->
+        <meta-data
+            android:name="com.google.ar.core"
+            android:value="optional" />
+
```

Comment:
Question : is this really needed ? The merged Manifest seems to indicate that it is already included, since SceneView already declares this `meta-data` tag (https://github.com/SceneView/sceneview-android/blob/main/arsceneview/src/main/AndroidManifest.xml#L8-L14).

Reply:
You're right, I will remove it